src/main/java/vectorwing/farmersdelight/integration/jei/category/DecompositionRecipeCategory.java

```diff
@@ -85,7 +90,9 @@ public void setIngredients(DecompositionDummy decompositionRecipe, IIngredients
 
 	@Override
 	public void setRecipe(IRecipeLayout recipeLayout, DecompositionDummy decompositionRecipe, IIngredients iIngredients) {
-		List<ItemStack> accelerators = ModTags.COMPOST_ACTIVATORS.getValues().stream().map(ItemStack::new).collect(Collectors.toList());
+		final Registry<Block> registry = (Registry<Block>) Registry.REGISTRY.get(ModTags.COMPOST_ACTIVATORS.registry().getRegistryName());
```

Comment:
This piece of code is based on Paradise Lost's RegistryHelper's methods, and should work fine, but verify this just in case

Reply:
Seems it was causing a soft exception in JEI. After doing some searching, I found this method to work:
```
List<ItemStack> accelerators = ForgeRegistries.BLOCKS.tags().getTag(ModTags.COMPOST_ACTIVATORS).stream().map(ItemStack::new).collect(Collectors.toList());
```

Reply:
There are **many** changes I'll have to do in JEI anyway, though, due to deprecations, so I can finish it up on my end.